jgiven-maven-plugin/build.gradle

```diff
@@ -43,14 +43,20 @@ generateMavenPlugin.configure {
     }
 }
 
-task copyMavenPlugin(type: Copy, dependsOn: generateMavenPlugin) {
+tasks.register("copyMavenPlugin", Copy){it->
+    it.dependsOn += generateMavenPlugin
+
     from('build/maven/target/classes') {
         include '**/*.xml'
+        exclude '**/*.class'
     }
     into 'build/classes/java/main'
+    javadoc.dependsOn += it
+    checkstyleMain.mustRunAfter(it)
 }
 
-task copyPom(type: Copy, dependsOn: generatePom) {
+tasks.register("copyPom", Copy){it
```

Comment:
it -> it.dependsOn ..?

Reply:
indeed...

Reply:
I guess this only works because groovy uses `it` as default argument, and we technically didn't specify one... .Ahh groovy